src/symmetricDifferenceWith.js

```diff
@@ -0,0 +1,31 @@
+var _curry3 = require('./internal/_curry3');
+var converge = require('./converge');
+var differenceWith = require('./differenceWith');
+var flip = require('./flip');
+var unionWith = require('./unionWith');
+
+
+/**
+ * Finds the set (i.e. no duplicates) of all elements contained in the first or second list, but not both.
+ * Duplication is determined according to the value returned by applying the supplied predicate to two list
+ * elements.
```

Comment:
Yes. In my own projects I enforce a strict 79-character limit for all lines, but I don't think Ramda should go this far. Wrapping prose is straightforward and non-controversial, though. :)